### PR TITLE
Add support for static IPv6 addresses for FreeBSD

### PR DESCRIPTION
**Pull request by einsibjarni**
Add IPv6 support for FreeBSD

```
Currently, FreeBSD ignores IPv6 addresses. This PR adds support for static IPv6 addresses

```

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/contributing.html)
 - [x] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: cloudinit/net/bsd.py
Comment (igalic) [inline cloudinit/net/bsd.py:149]: I'm surprised black or pylint don't complain about this

File: tests/unittests/distros/test_netconfig.py
Comment (igalic) [inline tests/unittests/distros/test_netconfig.py:384]: if anything in this file test file tests IPv6, i'd add bsd tests too

File: cloudinit/net/bsd.py
Comment (igalic) [inline cloudinit/net/bsd.py:111]: I see nothing for dynamic IPv6. Will this come in a new PR?

**Review by igalic — COMMENTED** [review]
❓

File: cloudinit/net/bsd.py
Comment (igalic) [reply cloudinit/net/bsd.py:149]: oh, this is the fixed version.

File: cloudinit/net/bsd.py
Comment (einsibjarni) [reply cloudinit/net/bsd.py:111]: We don't use any dynamic IPv6, so I left it out.

File: tests/unittests/distros/test_netconfig.py
Comment (einsibjarni) [reply tests/unittests/distros/test_netconfig.py:384]: I don't run any BSD other than FreeBSD. The changes to bsd.py where needed to get IPv6 working for FreeBSD, but it shouldn't cause any regression for others.

File: cloudinit/net/bsd.py
Comment (igalic) [reply cloudinit/net/bsd.py:111]: well it's still a very good start, thank you.

**Comment by igalic** [comment]
Integration tests are failing:

```
17.74s$ sudo snap install lxd

error: cannot perform the following tasks:

- Start snap "lxd" (23889) services (systemctl command [start snap.lxd.activate.service] failed with exit status 1: Job for snap.lxd.activate.service failed because the control process exited with error code.

See "systemctl status snap.lxd.activate.service" and "journalctl -xe" for details.

)

The command "sudo snap install lxd" failed and exited with 1 during .

Your build has been stopped.
```

I don't think you can do anything about that.
the Github actions aren't run, saying they are waiting for maintainer approval… which seems really weird.

File: tests/unittests/distros/test_netconfig.py
Comment (igalic) [reply tests/unittests/distros/test_netconfig.py:384]: that's not what I meant: I meant: if in this test_netconfig.py there are places that test IPv6, we should add tests for (Free)BSD there too.

Comment (einsibjarni) [reply tests/unittests/distros/test_netconfig.py:384]: Ah, I see what you mean. I'll see what I can do

Comment (einsibjarni) [reply tests/unittests/distros/test_netconfig.py:405]: https://github.com/canonical/cloud-init/pull/1839/commits/d59f7c8e6942437b8838222ae0418009f4e18d6b

**Review by igalic — APPROVED** [review]

**Comment by TheRealFalcon** [comment]
@einsibjarni PR looks good, but you signed the CLA and added your name to this as `einsibjani`, but your username is `einsibjarni`. Can you re-sign the CLA and update this PR with your correct username?

**Comment by einsibjarni** [comment]
> @einsibjarni PR looks good, but you signed the CLA and added your name to this as `einsibjani`, but your username is `einsibjarni`. Can you re-sign the CLA and update this PR with your correct username?

Sorry :) There's a long and funny story behind the two usernames, but I'll resign the CLA and update the PR

**Comment by einsibjarni** [comment]
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

The build has been terminated
```
I don't think I can trigger a rebuild except with a dummy commit, isn't that right?

**Review by TheRealFalcon — APPROVED** [review]
LGTM and verified CLA signature. Thanks!

**Pull request merged**
